### PR TITLE
Code refactoring

### PR DESCRIPTION
**Pull request by Ase020**

### CODE REVIEW

**Comment by McbullIan** [comment]
@JimmyTron is this the PR you need he;p reviewing?

**Comment by Ase020** [comment]
@sonylomo these are the changes that I made on the PR:

- Resized the podcastCards on the landing page
Screenshot 2023-11-29 090420

- Removed the Creative Team section on the About Us page
- Set the object-fit from cover to contain on the Event cards on the community page
- Deleted redundant files such as Headers and Footers

**Comment by McbullIan** [comment]
Hey @Ase020 . I've started reviewing your code and should be done by tomorrow. And could you please include a description in all your PRs moving forward?  It helps when it comes to reviewing your code and knowing exactly where I need to start.

**Review by JimmyTron — APPROVED** [review]

File: src/APP/pages/aboutUs/sections/PartnerCTA.jsx
Comment (collinskasyoki) [inline src/APP/pages/aboutUs/sections/PartnerCTA.jsx:1]: I am not sure if the commented parts will be implemented in the future or not. If not, they should be removed.

**Review by collinskasyoki — APPROVED** [review]

**Review by JimmyTron — COMMENTED** [review]
Lets remove anything that is commented out to keep the codebase clean

**Review by JimmyTron — CHANGES_REQUESTED** [review]
Resolve the conflicts then ill be able to merge

**Review by JimmyTron — APPROVED** [review]

**Review by JimmyTron — APPROVED** [review]

**Pull request closed without merging**